### PR TITLE
make new option templateStyleguideAdditionalStylesheets optional...

### PR DESCRIPTION
**Pull request by Jessman5**

### CODE REVIEW

**Comment by Jessman5** [comment]
the second commit fixed it. @mariusbuescher

**Comment by Jessman5** [comment]
I also re-added a much needed body class that was (accidentally?) removed.

File: generate/setup.js [hunk 1]
```diff
@@ -74,12 +74,16 @@ function style( options, callback ) {
...
 
+    var templateStyleguideAdditionalStylesheets = [];
+
     if ( Array.isArray( options.templateStyleguideAdditionalStylesheets ) ) {
         templateStyleguideAdditionalStylesheets = options.templateStyleguideAdditionalStylesheets;
     } else if ( typeof options.templateStyleguideAdditionalStylesheets === 'string' ) {
         templateStyleguideAdditionalStylesheets = [
             options.templateStyleguideAdditionalStylesheets
         ];
+    } else {
```
Comment (mariusbuescher) [inline generate/setup.js]: Not needed here. By default the variable is already an array.

**Review by mariusbuescher — CHANGES_REQUESTED** [review]

**Comment by Jessman5** [comment]
done.

**Review by mariusbuescher — APPROVED** [review]

**Comment by mariusbuescher** [comment]
Nice, thank you.

**Pull request merged**